.github/workflows/build-images.yml

```diff
@@ -7,15 +7,12 @@ jobs:
     runs-on: ubuntu-22.04
     environment: quay.io
     timeout-minutes: 30
-    # needs to use docker, as podman-built images are not accepted to RHEL 8's docker any more
```

Comment:
FTR, we don't need them in RHEL 8 any more, our infra now runs on Fedora CoreOS.